doc/download_model.md

```diff
@@ -0,0 +1,33 @@
+# Download Your Trained Model
+
+Once you have trained your model with SQLFlow you may need to download your
+model so that you can deploy some online service to do real-time prediction.
+
+To download your trained model from SQLFlow, you need to install our
+[command-line tool](run/cli.md) first. Then run the below command to download
+your model:
+
+```shell
+# Configure database connection string
+export SQLFLOW_DATASOURCE="mysql://root:root@tcp(127.0.0.1:3306)/?"
+# Configure sqlflow server address
+export SQLFLOW_SERVER="localhost:50051"
+# Assume you have trained a model named sqlflow_models.my_dnn_model
+./sqlflow get model "sqlflow_models.my_dnn_model"
```

Comment:
It would be easier for readers to understand the correlation if we include an SQLFlow TO TRAIN statement with the model name before this command-line.